test/uk/gov/hmrc/plasticpackagingtax/registration/views/ConfirmOrganisationBasedInUkViewSpec.scala

```diff
@@ -110,9 +110,10 @@ class ConfirmOrganisationBasedInUkViewSpec extends UnitViewSpec with Matchers {
 
         val form = ConfirmOrganisationBasedInUk
           .form()
-          .fillAndValidate(ConfirmOrganisationBasedInUk(None))
```

Comment:
This way of creating an invalid form results in an error message with key `unbind.failed` - presumably twirl/play generated